html/pfappserver/lib/pfappserver/Form/Config/Source/GoogleWorkspaceLDAP.pm

```diff
@@ -0,0 +1,93 @@
+package pfappserver::Form::Config::Source::GoogleWorkspaceLDAP;
+
+=head1 NAME
+
+pfappserver::Form::Config::Source::GoogleWorkspaceLDAP -
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Config::Source::GoogleWorkspaceLDAP
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+use pf::Authentication::Source::GoogleWorkspaceLDAPSource;
+extends 'pfappserver::Form::Config::Source::LDAP';
+our $META = pf::Authentication::Source::GoogleWorkspaceLDAPSource->meta;
+
+
+has_field client_certificate => (
```

Comment:
What's the difference between client_certificate and client_cert, it seems pretty confusing to have these 2 fields named that way